javascript/javascript1/week3/index.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>js-1-week-3 Homework</title>
+    <script src="arrayremoval.js"></script>
+    <script src="seriesduration.js"></script>
+    <script src="traveltimetask.js"></script>
+    <script src="notetakingapp.js"></script>
+    <script src="smartPhoneUsageApp.js"></script>
+</head>
+
+<body>
+    <h1>My Freecodecamp user name is Upasana_Pattnaik </h1>
+</body>
```

Comment:
Placing scripts at the bottom of the <body> element improves the display speed, because script interpretation slows down the display.
As the browser reads everything from top to bottom its best to place the html, then css and finally the javascript. Javascript is the functionality, so unless there is much need, place the <script> tag in the bottom of <body>